README.md

```diff
@@ -22,6 +22,8 @@ This example shows a central that scans for peripheral devices and then displays
 package main
 
 import (
+	"fmt"
```

Comment:
We try to avoid use of `fmt` unless really needed, in order to reduce size of the binary especially on embedded devices. That is the reason the examples are this way.